src/Account/Model/Account.php

```diff
@@ -16,25 +16,30 @@
  * @property AccountGeneralSettings $generalSettings
  * @property ShopCollection         $shops
  * @property int                    $status
+ * @property array                  $subscriptionFeatures
  */
 class Account extends Model implements StorableArrayable
 {
+    public const FEATURE_ORDER_NOTES = 'allow_order_notes';
+
     public $attributes = [
-        'id'              => null,
-        'platformId'      => null,
-        'contactInfo'     => ContactDetails::class,
-        'generalSettings' => AccountGeneralSettings::class,
-        'shops'           => ShopCollection::class,
-        'status'          => null,
+        'id'                   => null,
+        'platformId'           => null,
+        'contactInfo'          => ContactDetails::class,
+        'generalSettings'      => AccountGeneralSettings::class,
+        'shops'                => ShopCollection::class,
+        'status'               => null,
+        'subscriptionFeatures' => [],
     ];
 
     public $casts      = [
-        'id'              => 'int',
-        'platformId'      => 'int',
-        'contactInfo'     => ContactDetails::class,
-        'generalSettings' => AccountGeneralSettings::class,
-        'shops'           => ShopCollection::class,
-        'status'          => 'int',
+        'id'                   => 'int',
+        'platformId'           => 'int',
+        'contactInfo'          => ContactDetails::class,
+        'generalSettings'      => AccountGeneralSettings::class,
+        'shops'                => ShopCollection::class,
+        'status'               => 'int',
+        'subscriptionFeatures' => 'array',
```

Comment:
Graag als Collection, hoeft niet per se een custom collection te zijn als het een plat lijstje met strings is